rowan/src/Rowan-Services-Tests/JadeServerTest.class.st

```diff
@@ -58,15 +60,15 @@ JadeServerTest >> test_jadeServerHierarchyValid [
 	subclasses :=  jadeServerClass subclasses asArray.
 	self assert: subclasses size equals: 1. 
 	self assert: subclasses first name = #JadeServer64bit.
-	jadeServerClass := UserGlobals at: subclasses first name. 
+	jadeServerClass := Rowan globalNamed: subclasses first name. 
```

Comment:
These tests are designed, in part, to test that Rowan is properly setup. 

Therefore, instead of doing the class lookup through Rowan, I'd suggest we do it independent of Rowan. 
``System myUserProfile symbolList objectNamed: subclasses first name``

Thoughts?

Reply:
`Rowan globalNamed:` does the lookup using the standard GemStone mechanism which is equivalent to your suggestion and is the *correct* way to do global  lookup in Rowan:
1. [`Rowan class>>globalnamed:`](https://github.com/GemTalk/Rowan/blob/candidateV1.2/rowan/src/Rowan-Kernel/Rowan.class.st#L43):
   ```smalltalk
   ^ self platform globalNamed: aName
   ```
2. [`RwGsPlatform>>globalNamed:`](https://github.com/GemTalk/Rowan/blob/candidateV1.2/rowan/src/Rowan-GemStone-Core/RwGsPlatform.class.st#L101)
   ```smalltalk
   ^ Rowan image objectNamed: aString
   ```
1.[ `RowanGsImage class>>objectNamed:`](https://github.com/GemTalk/Rowan/blob/candidateV1.2/rowan/src/Rowan-GemStone-Core/RwGsImage.class.st#L396)
   ```smalltalk 
   ^ self symbolList objectNamed: aSymbol
   ```
2. [`RowanGsImage class>>symbolList`](https://github.com/GemTalk/Rowan/blob/candidateV1.2/rowan/src/Rowan-GemStone-Core/RwGsImage.class.st#L506)
   ```smalltalk
   ^ GsCurrentSession currentSession symbolList
   ```

Reply:
I am inclined to promote the use/habit of using `Rowan globalNamed` even in GemStone-specific code, because it is the portable way to do things ... and ensures that global lookup is done consistently throughout Rowan ... the subtle difference between your suggestion and the actual implementation in Rowan **is** indeed subtle but can lead to inconsistent results if not used consistently ... so I would prefer to use it **everywhere** rather than make a decision on a case by case basis which expression is appropriate ...

Reply:
I don't have strong opinions about this. I'm ok if you want to check this in. Let me know when you check in so I can integrate. 

As an aside - I might suggest adding a specific test in Rowan code to test ``Rowan globalNamed:`` if you haven't already done so.